content/_index.md

```diff
@@ -504,8 +504,8 @@ For historical reference, the APIs were versioned as `rX.Y.Z` where `X`
 roughly represents a breaking change, `Y` a backwards-compatible change, and
 `Z` a patch or insignificant alteration to the API.
 
-`v1.0` of Matrix was released on June 10th, 2019 with the following API
-versions:
+`v1.0` of Matrix was a symbolic version released on June 10th, 2019 with the
+following minimum API versions:
```

Comment:
I don't really think that "symbolic release" is going to mean much to the unfamiliar reader. Maybe:

```suggestion
The current global versioning system was introduced with `v1.1`. 
[Matrix 1.0][https://matrix.org/blog/2019/06/11/introducing-matrix-1-0-and-the-matrix-org-foundation/]
did not correspond directly to a specification version; instead, it was based on the following
versions for the individual APIs:
```

Reply:
Applied